webpack/assets/javascripts/react_app/common/helpers.js

```diff
@@ -6,4 +6,7 @@ export default {
     });
   },
   noop: Function.prototype, // empty function
+  getDisplayName(Component) {
+    return Component.displayName || Component.name || 'Component';
```

Comment:
isnt this the default (just with an Unknown string instead of Component?

Reply:
I'm not sure what are the defaults. I saw `displayName` being undefined at some places. This approach is a React.js convention for HOCs: https://reactjs.org/docs/higher-order-components.html#convention-wrap-the-display-name-for-easy-debugging